src/tokenizer/stop_word_filter.rs

```diff
@@ -0,0 +1,86 @@
+//! # Example
+//! ```
+//! extern crate tantivy;
+//! use tantivy::tokenizer::*;
+//!
+//! # fn main() {
+//! let tokenizer = SimpleTokenizer
+//!   .filter(StopWordFilter::remove(vec!["the".to_string(),"is".to_string()]));
+//!
+//! let mut stream = tokenizer.token_stream("the fox is crafty");
+//! assert_eq!(stream.next().unwrap().text, "fox");
+//! assert_eq!(stream.next().unwrap().text, "crafty");
+//! assert!(stream.next().is_none());
+//! # }
+//! ```
+use super::{Token, TokenFilter, TokenStream};
+
+/// `TokenFilter` that removes stop words from a token stream
+#[derive(Clone)]
+pub struct StopWordFilter {
+  words: Vec<String>,
```

Comment:
YAS, much better!

Reply:
Make sure to not use the default Hash too... The FnvHasher should be ok. In the future we will probably want to have something a tad smarter to avoid hashing tokens twice (once for StopWords and once to append to the posting lists.)

Reply:
Ok, added the `fnv` crate and now that is rocking.